core/src/main/java/org/apache/iceberg/CatalogUtil.java

```diff
@@ -117,6 +117,11 @@ public static void dropTableData(FileIO io, TableMetadata metadata) {
         Iterables.transform(metadata.previousFiles(), TableMetadata.MetadataLogEntry::file),
         "previous metadata",
         true);
+    deleteFiles(
+        io,
+        Iterables.transform(metadata.statisticsFiles(), StatisticsFile::path),
+        "statistic",
```

Comment:
```suggestion
        "statistics",
```

Reply:
thank you @ajantha-bhat , I used `statistic` earlier due to singular nouns here given how all existing code refer to file type, like `manifest`, `manifest list` and `previous metadata`. But I now recognize that the class name is already defined as `api/src/main/java/org/apache/iceberg/StatisticsFile.java`. let me update to your suggestion